bundle/manifests/activemq-artemis-operator.clusterserviceversion.yaml

```
@@ -224,6 +263,9 @@ metadata:
               "image": "placeholder",
               "size": 2
             }
+          },
+          "status": {
+            "podStatus": null
```

Could you use a condition with an empty array also in this example?
```suggestion
            "conditions": []
```